crates/cxx-qt-lib/src/assertion_utils.h

```diff
@@ -14,27 +15,39 @@
 namespace rust::cxxqtlib1::assertion_utils {
 template<typename iter>
 constexpr static ::std::size_t
-calc_align_size(const iter rbegin,
-                const iter rend,
+calc_align_size(const iter begin,
```

Comment:
Side-note: we should be able to avoid duplicating this in cxx-qt-lib-extras by simply doing an `#include <cxx-qt-lib/assertion_utils.h>` or something like this.

cxx-qt-lib-extras depends on cxx-qt-lib and reexports the headers.